rust/parquet/src/lib.rs

```diff
@@ -15,6 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#![feature(bufreader_seek_relative)]
```

Comment:
It's our intention to move to stable Rust at some point, we can revisit these in the future